web/react/components/delete_post_modal.jsx

```diff
@@ -40,6 +40,14 @@ export default class DeletePostModal extends React.Component {
         ModalStore.removeModalListener(ActionTypes.TOGGLE_DELETE_POST_MODAL, this.handleToggle);
     }
 
+    componentDidUpdate() {
+        if (this.state.show) {
```

Comment:
Two small things here:
1) I'd make this `if (this.state.show && !prevState.show) {` just to be a bit more consistent with how other modals do stuff on show
2) I'm not sure how setTimeout is works when there's no timeout specified. I'd change it to requestAnimationFrame instead